kedro/templates/project/{{ cookiecutter.repo_name }}/src/{{ cookiecutter.python_package }}/settings.py

```diff
@@ -1,8 +1,7 @@
 """Project settings."""
-from {{cookiecutter.python_package}}.hooks import ProjectHooks
```

Comment:
I think we should keep this line in, just commented out. Otherwise the line with `ProjectHooks()` is kind of mysterious because it's not clear where it comes from.

Reply:
Yes that makes sense.